src/handlers.ts

```diff
@@ -1,40 +1,53 @@
 import * as debug from 'debug';
 import * as semver from 'semver';
 
-import { REPOS, ROLL_TARGETS } from './constants';
+import { NUM_SUPPORTED_VERSIONS, REPOS, ROLL_TARGETS } from './constants';
 import { compareChromiumVersions } from './utils/compare-chromium-versions';
 import { getChromiumLkgr, getChromiumTags } from './utils/get-chromium-tags';
 import { getOctokit } from './utils/octokit';
 import { roll } from './utils/roll';
 
+// Get array of currently supported branches
+export async function getSupportedBranches(branches: any): Promise<string[]> {
+  const releaseBranches = branches.filter((branch) => {
+    const releasePattern = /[0-9]+-(?:(?:[0-9]+-x$)|(?:x+-y$))/;
```

Comment:
```suggestion
    const releasePattern = /^[0-9]+-([0-9]+-x|x-y)$/;
```

Since you're not using any of the captures, there's no need to worry about `(?:)` vs `()`. Plus some other cleanups, to avoid matching `foo-8-x-y` and `8-xxxxx-y`